ts/modules/networkgraph/QuadTree.ts

```diff
@@ -151,6 +139,8 @@ H.extend(
             point: Highcharts.NetworkgraphPoint,
             depth: number
         ): void {
+            var newQuadTreeNode: Highcharts.QuadTreeNode;
```

Comment:
FYI: `const` and `let` can be used in our TypeScript files and is transpiled nicely to `var` in our JS files. Block scoping is also handled nicely, which is relevant for this example where `newQuadTreeNode` is used only within the `else` block.

Reply:
Thanks for the info!